app/models/daily_habit.rb

```diff
@@ -0,0 +1,10 @@
+class DailyHabit < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :habit
+
+  scope :completed_today, -> { where(
+    'created_at > :beg AND created_at < :end',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.